AdyenCard/Components/Card/CardComponent.swift

```diff
@@ -222,6 +222,12 @@ public class CardComponent: PresentableComponent,
             type: data.type
         )
         infoEvent.target = data.target
+        infoEvent.brand = data.brands?.first?.type.rawValue
+        if data.type == .selected {
```

Comment:
This logic is very obscure. Let's either use additional variables/methods named in a way that makes it easy to understand?